test/shared/extends.js

```diff
@@ -0,0 +1,29 @@
+var cons = require('../../')
+  , fs = require('fs');
+
+var should = require('should');
+
+exports.test = function(name) {
+  var user = { name: 'Tobi' };
+
+  describe(name, function(){
+
+    if (name == 'nunjucks') {
```

Comment:
Is it fine to make it as option to `includes` specs? I've just updated to such solution, let me know if you meant something else.